h/test/tweens_test.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from pyramid.testing import DummyRequest as _DummyRequest
+from h import tweens
+
+
+class DummyRegistry():
+    def __init__(self, *args, **kwargs):
+        self.settings = {}
+
+
+class DummyRequest(_DummyRequest):
+    def __init__(self, *args, **kwargs):
+        self.registry = DummyRegistry()
+        super(DummyRequest, self).__init__(*args, **kwargs)
+
+
+class DummyHeader():
```

Comment:
Oh I see, you need the response object. So again, perhaps use a real request/response object:

```python
handler = lambda request: request.response
```